model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "[Request charge](https://learn.microsoft.com/azure/cosmos-db/request-units) consumed by the operation"
+    instrument: histogram
+    unit: "{request_unit}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.client.active_instances
+    type: metric
+    metric_name: db.cosmosdb.client.active_instances
```

Comment:
since it sounds like we have a mini-consensus, should we go ahead in this PR and start to follow 
https://github.com/open-telemetry/semantic-conventions/issues/1494#issuecomment-2422829241?

Reply:
Also, it'd be nice to follow existing naming guidelines https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#use-count-instead-of-pluralization-for-updowncounters

in particular "Use count Instead of Pluralization for UpDownCounters" - suggesting `db.cosmosdb.active_client.count` or 
`db.cosmosdb.client.active_instance.count`

Reply:
actually I took this from https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpclientactive_requests
https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpserveractive_requests
https://opentelemetry.io/docs/specs/semconv/dotnet/dotnet-kestrel-metrics/#metric-kestrelactive_connections
https://opentelemetry.io/docs/specs/semconv/dotnet/dotnet-kestrel-metrics/#metric-kestrelactive_tls_handshakes

Similarly, I included this metric. I believe we might be over-complicating things here (if I may say so).

Reply:
the fact that we didn't follow the guidance in several places should not be an excuse to not follow it. There are exceptions from every guidance, but there are way more metrics where we follow the guidance.

Reply:
I wasn't offering an excuse; rather, I was highlighting that this is a convention commonly followed in the documentation. However, as you pointed out it wasn't used consistently before, I've now made the necessary changes.